fr/taxonomie_utilisateurs.md

```diff
@@ -0,0 +1,14 @@
+Taxonomie des utilisateurs
+==========================
```

Comment:
But I'd rather say: if it works with Markdown, *and* we don't have linting tools, don't add cognitive overload!